packages/slonik/src/helpers.test/createIntegrationTests.ts

```diff
@@ -968,8 +968,6 @@ export const createIntegrationTests = (
 
     await pool.end();
 
-    await delay(600);
-
```

Comment:
This `delay` was actually masking this issue. What was happening was that `pool.end()` didn't actually close all connections like it was supposed to, we were simply waiting for the `idleTimeout` to expire so the connections automatically get closed.

Removed the `delay` and set `idleTimeout: 'DISABLE_TIMEOUT'` so that if we break this functionality the test should timeout.